src/shared/coredump-util.h

```diff
@@ -22,6 +22,9 @@ typedef enum CoredumpFilter {
                                       1u << COREDUMP_FILTER_ELF_HEADERS | \
                                       1u << COREDUMP_FILTER_PRIVATE_HUGE)
```

Comment:
maybe change `1u` → `UINT32_C(1)` at the same time?

Reply:
let's leave that for another day, doesn't make a difference anyway